docs/tutorials/05_animation/viz_keyframe_bezier_interpolator.py

```diff
@@ -28,33 +28,41 @@
 ###############################################################################
```

Comment:
- in this tutorial, I always encounter a segmentation fault when I try to run the complex scene.  it seems, that it works perfectly when I remove `showm.initialize()`. Can you remove `showm.initialize()`everywhere.

- Complex scene title has a typo (scene Twice instead of one)

Reply:
It works and renders only one time , but time callback functions do not work in case of not initializing